ts/features/idpay/initiative/details/screens/InitiativeDetailsScreen.tsx

```diff
@@ -105,10 +112,13 @@ export const InitiativeDetailsScreen = () => {
   useFocusEffect(
     useCallback(() => {
       dispatch(idpayInitiativeGet.request({ initiativeId }));
+      dispatch(idpayTimelinePageGet.request({ initiativeId, page: 0 }));
     }, [dispatch, initiativeId])
   );
 
   const isLoading = pot.isLoading(initiativeDetailsFromSelector);
+  const isFirstInitiativeConfiguration =
+    useIOSelector(idpayTimelineSelector).length <= 1;
```

Comment:
```suggestion
  const timelineSize = useIOSelector(idPayTimelineSizeSelector);
  const isFirstInitiativeConfiguration = timelineSize <= 1;
```

Maybe an `idPayTimelineSizeSelector` could be more efficient?

Reply:
kind of a forgettable change to be honest, more readable nonetheless and adressed in https://github.com/pagopa/io-app/pull/4444/commits/b8523ccad48dd4aeeaded7a7c044d9b6e8f384ef

Reply:
We need to make sure everyone understands the code we write. **One-liners are bad**. If you like it, play some [code golf ](https://code.golf/)😆